tests/GithubIntegration.py

```diff
@@ -61,6 +61,27 @@ def testCreateJWT(self):
         )
         sys.modules["time"].time = self.origin_time
 
+    def testCreateJWTWithExpiration(self):
+        self.origin_time = sys.modules["time"].time
+        sys.modules["time"].time = lambda: 1550055331.7435968
```

Comment:
This feels awkward. Why not use a mock here?

Reply:
I don't like it either, just copied from existing `testCreateJWT` above.

Reply:
Then we should fix both places, and I'd rather circle back and fix one, rather than both.

Reply:
I don't have time to cut a release right now, but I'll get this merged.